vm-image-service/README.md

```diff
@@ -49,14 +27,37 @@ The UI allows users to execute all necessary workflows:
         - Show details for the Content Library
     - Grid action menu
         - View the YAML for a Content Library
+- Publish VM to a Content Library
 
 ## List all Virtual Machine Images
+![VM Image Card](source/images/vm-image-card.png "VM Image Card")
+VM Image service can be accessed via a card on the namespace summary view. If VM Image service is available for the current namespace then the above shown card will appear in the namespace summary view. Check out the support section in this document.
+
+![VM Image List View](source/images/vm-image-image-list.png "VM Image List View")
+In the above picture, we are looking at the list of VirtualMachine Images which are available for the current namespace. Namespaces are not directly associated with VM Images rather transitively associated through Content Libraries. The above list view is an aggregation of all the VM Images across all Content Libraries that are associated with the current namespace.
 
 ### Actions
+![VM Image Delete](source/images/vm-image-delete.png "VM Image Delete")
+The only user action available for consumption users to perform on a VM Image is "Delete" a VM Image from a namespace-scoped Content Library that has been marked as writable. Delete action will be left unavailable for VM Images which belong to cluster-scoped and read-only libraries.
+
 ### Details
+![VM Image Summary](source/images/vm-image-summary.png "VM Image Summary")
+Clicking an image name in the VM Image list view will navigate to a detailed summary view for the selected VM Image. On this view, you will be able to view all the details of VM Image, check health status, and view the list of latest Kubernetes events for the given image.
```

Comment:
of VM Image => of the VM Image

Reply:
what's the "health" of an image? Can you explain this a bit more?

Reply:
> what's the "health" of an image? Can you explain this a bit more?

On CCI, we display resources' "status.conditions" in a table on the respective resource Summary page. We name the section as "<Resource name> Health Status", in this case the section is named "Image Health Status". 

https://onevmw-my.sharepoint.com/:i:/g/personal/somashekars_vmware_com/EUtgnXo3zHNAmpfSBpHf-0oBSWoPHbLyRHoZR725spPZGA?e=oaW153